Remotion/Web/Core/UI/Controls/WebButtonType.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Remotion.Web.UI.Controls
+{
+  /// <summary>
+  /// Defines how a <see cref="WebButton"/> is displayed on the page.
+  /// </summary>
+  public enum WebButtonType
+  {
+    /// <summary>
+    /// The <see cref="WebButton"/> is displayed using the standard style.
+    /// </summary>
+    Standard,
+
+    /// <summary>
+    /// The <see cref="WebButton"/> is displayed using a style indicating a primary action.
+    /// </summary>
+    Primary,
+
+    /// <summary>
+    /// The <see cref="WebButton"/> is displayed using a style indicating a secondary action.
```

Comment:
Given that `Standard` is used next to `Primary` on the page, maybe we should get a better name for `Secondary`. Problem is, same as during the design of this task, I don't have a good name. `Extra`, `Additonal`, or `Other`. Ideas / Thoughts?

Reply:
Bootstrap calls it a `link` button since it does not have a button outline

Reply:
yeah, I can see why, but that doesn't make it better when used with the dropdownmenu. It also makes an assumption on the look&feel which we shouldn't because the no-outline-thing could change in a future theme.

Reply:
`supplemental` it is